### DISCUSSION TITLE
微软双拼键盘缺少分号

### DISCUSSION THREAD

**Discussion by ChinaYong** [fcitx5-android/fcitx5-android#330] [Q&A | 问答]
微软双拼ing怎么打出来？Gboard会把大写锁定的键变成分号，但我用了好几天都没捣鼓出来这款软件怎么打出ing

**Comment by rocka** [comment]
TLDR: 建议换成自然码双拼

https://github.com/fcitx5-android/fcitx5-android/issues/73

**Reply by tiffanyu** [reply]
微软双拼暂时可以用z那里的‘代替,但是如果韵母是在末尾会有一些问题,不完美。自然码用y可以打出ing。新版本期待大佬可以修复下。
